content/en/blog/_posts/2017-08-00-Kubernetes-Meets-High-Performance.md

```diff
@@ -3,6 +3,8 @@ title: " Kubernetes Meets High-Performance Computing "
 date: 2017-08-22
 slug: kubernetes-meets-high-performance
 url: /blog/2017/08/Kubernetes-Meets-High-Performance
+author: >
+   Robert Lalonde (Univa) 
 ---
 Editor's note: today's post is by Robert Lalonde, general manager at Univa, on supporting mixed HPC and containerized applications &nbsp;
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.